src/Provider.js

```diff
@@ -61,9 +63,13 @@ class Provider extends Component {
     alert.close = () => this.remove(alert)
 
     if (alert.options.timeout) {
-      setTimeout(() => {
+      let timerId = setTimeout(() => {
```

Comment:
We don't need to mutate the `timerId` id later, so we can declare it as a `const`, like `const timerId = `.

Reply:
Changed to const